.gitignore

```diff
@@ -38,6 +38,12 @@ obj/
 *.log
 *.idb
 *.opendb
+
 WSATestCertificate.pfx
 WSATestCertificate.pfx.meta
+
+# ============================= #
+# Custom Project Specific Paths #
+# ============================= #
+Assets/AssetStoreTools*
```

Comment:
Does this line (and the line after) need to be in the repo? It seems like this will allow people to commit their own specific folders to be ignored, even though nobody else will have them.

Reply:
Agreed. Not sure why it was here to begin with

Reply:
Yes to the line below, no to the current line